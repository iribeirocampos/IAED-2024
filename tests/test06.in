p parque1 5 0.30 0.50 15.00
p "Parque 2" 5 0.20 0.40 13.00
e parque1 AA-00-AA 01-01-2024 8:00
e parque1 AA-01-AA 01-01-2024 8:01
e parque1 AA-02-AA 01-01-2024 8:02
e parque1 AA-03-AA 01-01-2024 8:03
e parque1 AA-04-AA 01-01-2024 8:03
e parque3 AA-14-AA 01-01-2024 8:03
e parque1 AA-15-AA 01-01-2024 8:03
e "Parque 2" AA-06-AA 01-01-2024 8:14
e "Parque 2" AA-07-AA 01-01-2024 8:15
e "Parque 2" AA-08-AA 01-01-2024 8:16
e "Parque 2" AA-09-AA 01-01-2024 8:16
e "Parque 2" AA-AA-AA 01-01-2024 8:20
e "Parque 2" 46-87-82 01-01-2024 8:21
e "Parque 2" 46-AA-82 01-01-2024 8:00
e "Parque 2" 46-AA-82 31-02-2024 8:00
e "Parque 2" 46-AA-82 31-06-2024 8:00
e "Parque 2" 46-AA-82 32-07-2024 8:00
e "Parque 2" 46-AA-82 28-00-2025 8:00
e "Parque 2" AA-01-AA 02-01-2024 8:00
p
q
