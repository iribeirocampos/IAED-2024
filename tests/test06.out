parque1 4
parque1 3
parque1 2
parque1 1
parque1 0
parque3: no such parking.
parque1: parking is full.
Parque 2 4
Parque 2 3
Parque 2 2
Parque 2 1
AA-AA-AA: invalid licence plate.
46-87-82: invalid licence plate.
invalid date.
invalid date.
invalid date.
invalid date.
invalid date.
AA-01-AA: invalid vehicle entry.
parque1 5 0
Parque 2 5 1
